.travis.yml

```diff
@@ -25,6 +26,7 @@ addons:
       - liblzma-dev  # xv
       - libzstd-dev
       - zlib1g-dev
+      - libssl-dev  # needed for CMake to download deps
```

Comment:
This probably isn't needed now that we have pre-built binary?

Reply:
I will remove it